views/scss/inc/_test-layout.scss

```diff
@@ -91,6 +91,11 @@
             bottom: 0;
             width: 100%;
             opacity : .9;
+
+            &-full {
+                background-color: Window;
```

Comment:
By default, the background is transparent and the opacity is not taken into account unless a solid background color is set. This line just arbitrary force the background color with the window's one.

The main concern is about which color to choose, as the custom theme may overwrite it. Taking the color of the window was a mere solution.